BaseFilter/sections/css_extended.txt

```diff
@@ -1946,6 +1946,7 @@ indiastudychannel.com#?##content_rightbar > div.roundedBlock:has(> div#HP-gpt-pa
 indiastudychannel.com#?##content_rightbar > div.roundedBlock:has(> div[id^="div-gpt-ad-"])
 !
 linuxadictos.com#?##content > aside[id^="text-"]:has(> div.textwidget > p.gt-block > ins.adsbygoogle)
+techpowerup.com#?#.v-port > div:has(> a[rel="nofollow"][href^="/reviyuu/"])
```

Comment:
Where is this rule needed?

Reply:
I think the website design have changed.
so, the rule has to removed.